examples/platform/silabs/SiWx917/SiWx917/sl_wifi_if.cpp

```diff
@@ -262,23 +260,36 @@ sl_status_t join_callback_handler(sl_wifi_event_t event, char * result, uint32_t
 
 #if CHIP_CONFIG_ENABLE_ICD_SERVER
 #if SLI_SI91X_MCU_INTERFACE
-// Required to invoke button press event during sleep as falling edge is not detected
-void sl_si91x_invoke_btn_press_event(void)
+// This function is called when the button is pressed when in sleep then the idleTask is handling the button press event
```

Comment:
Can you adapt these comments to the appropriate format?
This comment applies to all functions.

```
/**
 * @brief      invoked when enhanced connection complete event is received
 * @param[out] resp_conn, connected remote device information
 * @return     none.
 */
```

Can you add to the comment that the signature is important since it is a weak function.

Reply:
Again, resolved but not addressed.

Reply:
@mkardous-silabs I have made the changes for them, was looking for the input on the header file, before committing it here.